contracts/Pod.sol

```diff
@@ -114,7 +114,7 @@ contract Pod is Initializable, ReentrancyGuardUpgradeSafe, ContextUpgradeSafe, B
 
     function operatorRedeemSharesInstantly(address _from, uint256 _shares) external nonReentrant returns (uint256) {
         address _operator = _msgSender();
-        require(podShares.isOperatorFor(_operator, _from), "Pod: Invalid operator");
+        require(podShares.allowance(_from, _operator) >= _shares, "Pod/exceeds-allowance");
```

Comment:
Remember you have to decrease their allowance as well!

Have a look at the ControlledToken code in PT.  It includes the function:

```solidity
  function controllerBurnFrom(address _operator, address _user, uint256 _amount) external virtual onlyController {
    if (_operator != _user) {
      uint256 decreasedAllowance = allowance(_user, _operator).sub(_amount, "ERC20: burn amount exceeds allowance");
      _approve(_user, _operator, decreasedAllowance);
    }
    _burn(_user, _amount);
  }
```

Using this burn function by passing the operator means that we don't need any additional checks; it's captured here *and* it also updates the allowance.

Additionally, since we're not using ERC777 anymore we can remove the "operator*" functions and just have a single function for users to redeem.  They just pass themselves if they wish to redeem for themselves.